include/jwt-cpp/traits/boost-json/defaults.h

```diff
@@ -25,9 +25,12 @@ namespace jwt {
 	}
 
 	/**
-	 * Return a builder instance to create a new token
+	 * Create a builder using the default clock
+	 * \return builder instance to create a new token
 	 */
-	inline builder<traits::boost_json> create() { return builder<traits::boost_json>(); }
+	inline builder<default_clock, traits::boost_json> create() {
+		return builder<default_clock, traits::boost_json>(default_clock{});
+	}
```

Comment:
This also needs to be in the template 
https://github.com/Thalhammer/jwt-cpp/blob/953aab6fe47be9e92fcbf5fb8418dee0d682e7ca/include/jwt-cpp/traits/defaults.h.mustache#L32

Reply:
Done